src/components/List.js

```diff
@@ -101,42 +92,22 @@ const List = ({ token }) => {
     <div>
       <h1>List</h1>
 
-      {docs.length === 0 ? (
+      {docs.length === 0 && (
         <section>
           <p>Your shopping list is currently empty.</p>
           <a href="/add-item">Add an Item</a>
         </section>
-      ) : (
-        <div>
-          <label htmlFor="search-bar">Filter Items</label>
-          <br />
-          <input
-            type="text"
-            name="search-bar"
-            id="search-bar"
-            placeholder="Start typing here..."
-            value={searchInput}
-            onChange={handleSearchChange}
-          />
-          <input type="reset" onClick={handleClear} />
-        </div>
       )}
 
       {errorMessage && <Error errorMessage={errorMessage} />}
 
       <ul style={{ listStyleType: 'none' }}>
         {docs &&
-          docs
-            ?.filter((doc) =>
-              doc?.itemName
-                ?.toLowerCase()
-                ?.includes(searchInput.toLowerCase().trim()),
-            )
-            ?.map((doc) => (
+          docs.map((doc) => {
+            return (
               <li key={doc.id}>
                 <input
                   type="checkbox"
-                  aria-label="purchased-checkbox"
```

Comment:
I noticed I don't have this line. Should we put it back in? Looks like it will now be on line 140.

Reply:
I added this for accessibility, so yes if we could put it back in.